src/test/java/de/tum/in/www1/artemis/post/ConversationUtilService.java

```diff
@@ -258,13 +310,28 @@ private List<Post> createBasicPosts(Course courseContext, CourseWideContext[] co
         return posts;
     }
 
+    /**
+     * Creates and saves a Post for the given PlagiarismCase.
+     *
+     * @param plagiarismCase The PlagiarismCase the Post belongs to
+     * @param userPrefix     The prefix of the author's login the Post belongs to (the login is appended with "instructor1")
+     * @return The created Post
+     */
     public Post createBasicPost(PlagiarismCase plagiarismCase, String userPrefix) {
         Post postToAdd = ConversationFactory.createBasicPost(0, userUtilService.getUserByLoginWithoutAuthorities(String.format("%s%s", userPrefix + "instructor", 1)));
         postToAdd.setPlagiarismCase(plagiarismCase);
         postToAdd.getPlagiarismCase().setExercise(null);
         return postRepository.save(postToAdd);
     }
 
+    /**
+     * Creates and saves 3 Posts for the given Conversation.
```

Comment:
Maybe you could elaborate on this a bit more. What are those 3 posts and why are they generated?

Reply:
I tried to be more descriptive and updated the documentation.